compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/ExtendedDeclarationCheckers.kt

```diff
@@ -37,4 +38,9 @@ object ExtendedDeclarationCheckers : DeclarationCheckers() {
         get() = setOf(
             RedundantReturnUnitType,
         )
+
+    override val regularClassCheckers: Set<FirRegularClassChecker>
+        get() = setOf(
+            SafeInitialisationChecker            
```

Comment:
I'd recommend to make an experiment by adding this checker also to `CommonDeclarationCheckers` and run all tests after that.